### ISSUE TITLE
Unable to execute request: not connected to a Kuzzle server.

### ISSUE THREAD

**Issue by davidmoshal** [kuzzleio/kuzzle#1703]
## Expected Behavior

kuzzle-sdk should be able to connect to kuzzle server

## Current Behavior

Given:

- a kuzzle server set up as described in the docs,

- running on http://localhost:7512
  - verified by response from browser

- and running on ws://localhost:7512  
  - verified using Chrome Extension: SimpleWebsocketClient

Then:

- the first example in the getting started docs fails
  - https://docs.kuzzle.io/core/2/guides/getting-started/first-steps/

```js
const {
  Kuzzle,
  WebSocket
} = require('kuzzle-sdk');

const kuzzle = new Kuzzle(new WebSocket('localhost'));

try {
  // Create a 'playground' index
  await kuzzle.index.create('playground');
  // Create a collection named 'mycollection' in playground index
  await kuzzle.collection.create(
    'playground',
    'mycollection'
  );
  console.log('playground/mycollection ready');
} catch (error) {
  console.error(error.message);
}
```
 
In other words, I can connect to the Kuzzle server using chrome on http and ws, but I cannot connect to it from the sdk?

**Comment by davidmoshal** [comment]
Solved, it's a documentation bug.
After the line:
```js
const kuzzle = new Kuzzle(new WebSocket('localhost'));
```
Insert:
```js
await kuzzle.connect
```

**Comment by Aschen** [comment]
Thanks for the feedback! I will fix the documentation.
